cors.py

```diff
@@ -0,0 +1,10 @@
+from flask import Flask
```

Comment:
I think "make_response" has to be explicitly imported into the working namespace like so:
`from flask import Flask, make_response`